xc/domain.ml

```diff
@@ -254,8 +254,14 @@ let make ~xc ~xs vm_info vcpus domain_config uuid final_uuid =
     flags = flags;
     max_vcpus = vcpus;
     max_evtchn_port = -1;
```

Comment:
Should this be configurable as well?

Reply:
No, we only allow to configure granttable limits for now - it mirrors master commit to 4.11 fro Ross